### PR TITLE
adding vscode run & debug config

### PR DESCRIPTION
**Pull request by meherhendi**
This adds the config for backend for anyone to run the debugger easily from vscode

### CODE REVIEW

**Comment by yuhongsun96** [comment]
Hey, this is a good idea, thanks for the contribution!

Would you mind also having the setup work for the background indexing process and the file connector process?

You can refer to https://github.com/danswer-ai/danswer/blob/main/backend/supervisord.conf

**Comment by meherhendi** [comment]
All background services in [ https://github.com/danswer-ai/danswer/blob/main/backend/supervisord.conf]( https://github.com/danswer-ai/danswer/blob/main/backend/supervisord.conf) were added,
Thanks for the feedback!

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        {
+            "name": "Danswer slack bot listener",
+            "type": "python",
+            "request": "launch",
+            "program": "danswer/listeners/slack_listener.py",
+            "cwd": "${workspaceFolder}/backend",
+            "env": {
+                "LOG_LEVEL": "DEBUG",
+                "PYTHONPATH": ".",
+                "DANSWER_BOT_SLACK_APP_TOKEN": "xapp-xxxxxxxxxxxxxxx",
```
Comment (Weves) [inline .vscode/launch.json]: IMO, these should be set in the `.env` file so we don't need to modify a checked in file to get this running. Can we just make these comments which tell the user to set these in a `.env` file located at the root of the project, and then specify `"envFile": "${workspaceFolder}"`?

**Review by Weves — CHANGES_REQUESTED** [review]
Thanks for this PR, this is quite useful! Added one quick comment, but other than that it lgtm

File: .vscode/launch.json (hunk 1 above)
Comment (meherhendi) [reply .vscode/launch.json]: I agree, this would risk leaking the API keys, 
I think adding an .env.example file in a future PR would be a good idea too.

Comment (Weves) [reply .vscode/launch.json]: We do have some template `.env` files in the `deployment` directory. Check those out and see if they fit your needs 🙏

**Review by Weves — APPROVED** [review]

**Comment by Weves** [comment]
Thanks for this change 🙇

**Pull request merged**